lib/resources/dstu2/condition.yaml

```diff
@@ -71,6 +72,15 @@ fields:
                    "display": "Generalized nonconvulsive epilepsy",
                    "userSelected": true}],
                 "text": "Absence seizures"}}
+  binding:
+    description: A code that defines the condition.
+    terminology:
+    - display: SNOMED CT
+      system: http://snomed.info/sct
+    - display: ICD-9
+      system: http://hl7.org/fhir/sid/icd-9
```

Comment:
Agreed that 'http://hl7.org/fhir/sid/icd-9' is not listed in terminologies system. But this is currently what we are returning in response and the fhir page it links to provides source of this code system.